src/web/core/src/main/resources/GeoServerApplication_nl.properties

```diff
@@ -1,32 +1,33 @@
 AboutGeoServerPage.documentation = Documentatie
 AboutGeoServerPage.geoserverVersion = Versie
 AboutGeoServerPage.geotoolsVersion = GeoTools
 AboutGeoServerPage.geowebcacheVersion = GeoWebCache
-AboutGeoServerPage.information = De documentatie voor deze versie is beschikbaar op het internet via de volgende hyperlink. De GeoServer wiki bevat de meest recente informatie; gebruik de wiki om ervaringen, tips en uitleg te delen. Het meldingensysteem is de plaats om fouten en wensen te rapporteren. 
+AboutGeoServerPage.information   = De beheeromgeving is bedoeld om GeoServer eenvoudig te kunnen beheren. Na het aanmelden kunnen de menu's aan de linkerzijde worden gebruikt om te navigeren. "Over GeoServer" en "Server Status" tonen informatie over de geinstalleerde GeoServer. Het "Configuratie" menu biedt mogelijkheden om bronnen in te stellen en stijlen aan te maken. Het "Services" menu geeft de mogelijkheid de configuratie van de services in te stellen. Onder "Instellingen" kan de technische systeem configuratie worden ingesteld. Het "Tegels" menu is bedoeld om GeoWebCache in te stellen. Onder "Beveiliging" kunnen toegang en rechten worden ingesteld. Bij "Demo's" kunnen de mogelijkheden van GeoServer worden uitgeprobeerd en onder "Gereedschappen" vind je een aantal handige hulpmiddelen.
+
 AboutGeoServerPage.moreInformation = Informatie
- 
+
 # see comment in AboutGeoServerPage userMap, add the following lines at the end of
 # AboutGeoServerPage.information property
-# Also please take a moment to add \
-# yourself to the User Map to show your support for GeoServer.
+# Also please take a moment to add # yourself to the User Map to show your support for GeoServer.
+
+AboutGeoServerPage.introduction  = GeoServer is een Java implementatie van de Web Feature Service (WFS), Web Coverage Service (WCS) en Web Map Services (WMS) conform de door het Open Geospatial Consortium (OGC) gedefinieerde standaarden.
 
-AboutGeoServerPage.introduction = GeoServer is een transactionele Java (J2EE) implementatie van de \
-    OpenGIS Consortium's WFS en WCS specificatie, met een geintegreerde WMS.
 AboutGeoServerPage.gitRevision = Revisie
-AboutGeoServerPage.title = Over GeoServer
-AboutGeoServerPage.userMap = Kaart van gebruikers
-AboutGeoServerPage.wiki = Wiki
+AboutGeoServerPage.title         = Over GeoServer
+AboutGeoServerPage.usefulLinks   = Handige informatie\:
+AboutGeoServerPage.userMap       = Kaart van gebruikers
+AboutGeoServerPage.wiki          = Wiki
 
 AbstractCoverageStorePage.dataSrcName = Bron
 AbstractCoverageStorePage.description = Omschrijving
 
 AbstractDataAccessPage.cantGetDataStoreFactory = Kan geen plugin vinden voor de opgegeven bron. Controleer of de benodigde plugin is geinstalleerd
-AbstractDataAccessPage.dataSrcName = Bron
-AbstractDataAccessPage.description = Omschrijving
+AbstractDataAccessPage.dataSrcName             = Bron
+AbstractDataAccessPage.description             = Omschrijving
```

Comment:
The above looks wrong, as it only provides whitespace changes - and those whitespace changes may be a tab? Is this change intentional?